### PR TITLE
Add example for diverging color

### PR DESCRIPTION
**Pull request by YuhanLu**
Provide example for #449 
Modified doc for scale to include this example

### CODE REVIEW

File: examples/specs/diverging_color_points.vl.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "description": "A bubbleplot showing horsepower on x, miles per gallons on y, and binned acceleration on size.",
```
Comment (kanitw) [inline examples/specs/diverging_color_points.vl.json]: description doesn't match

File: examples/specs/diverging_color_points.vl.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "data": {"url": "data/cars.json"},
+  "mark": "point",
+  "encoding": {
+    "x": {"field": "Horsepower","type": "quantitative"},
+    "y": {"field": "Miles_per_Gallon","type": "quantitative"},
+    "color": {
+      "field": "Weight_in_lbs",
+      "type": "linear",
+      "scale": {
+        "domain": [1100,2700,5200],
```
Comment (kanitw) [inline examples/specs/diverging_color_points.vl.json]: How about ...

"domain": [1000,2750,4500],

(more rounded number)

File: site/docs/scale.md [hunk 3]
```diff
@@ -186,6 +186,10 @@ For ordinal, quantitative, and time fields, `range` can be a two-element array d
 
 <div class="vl-example" data-name="scatter_color_ordinal_custom"></div>
 
+We can also create diverging color graph by specify `range` with multipe elements.
```
Comment (kanitw) [inline site/docs/scale.md]: 1) typo `multipe` => `multiple`

2) Maybe add 
```
### Piecewise Scale Example: Diverging Color Scale
```
3) in the example you also have to specify `domain` to match `range`, but that is not described here. 

Maybe look at how Mike describe about piece wise case there so people understand the constraint better:
https://github.com/d3/d3-scale#continuous_domain

**Comment by YuhanLu** [comment]
The previous requests have been done.

File: examples/specs/diverging_color_points.vl.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "description": "A bubbleplot showing horsepower on x, miles per gallons on y, and binned weight of diverging color.",
```
Comment (kanitw) [inline examples/specs/diverging_color_points.vl.json]: binned weight?  I don't see `bin` in the spec.

**Pull request merged**